components/ntp_sponsored_images/browser/ntp_sponsored_images_component_installer.cc

```diff
@@ -0,0 +1,140 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/ntp_sponsored_images/browser/ntp_sponsored_images_component_installer.h"
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "base/base64.h"
+#include "base/bind.h"
+#include "base/callback.h"
+#include "brave/components/brave_ads/browser/locale_helper.h"
+#include "brave/components/ntp_sponsored_images/browser/ntp_sponsored_images_service.h"
+#include "brave/components/ntp_sponsored_images/browser/regional_component_data.h"
+#include "brave/vendor/bat-native-ads/src/bat/ads/internal/locale_helper.h"
```

Comment:
why are we getting the internal one for bat-native-ads vs brave/components/brave_ads/browser/locale_helper.h ?

Reply:
this header shouldn't even be available to use here